var/spack/repos/builtin/packages/py-pennylane-lightning-kokkos/package.py

```diff
@@ -34,17 +36,21 @@ class PyPennylaneLightningKokkos(CMakePackage, PythonExtension, CudaPackage, ROC
     for backend in backends:
         deflt_bool, descr = backends[backend]
         variant(backend.lower(), default=deflt_bool, description=descr)
-        depends_on(f"kokkos+{backend.lower()}", when=f"+{backend.lower()}", type=("run", "build"))
+        depends_on(
+            f"kokkos@:3.7.01+{backend.lower()}", when=f"+{backend.lower()}", type=("run", "build")
```

Comment:
I think we don't want `kokkos<3.7`, no?

```suggestion
            f"kokkos@3.7+{backend.lower()}", when=f"+{backend.lower()}", type=("run", "build")
```

Reply:
We have working with 3.6 in the past, but fine to fix this at 3.7 for now until 4.0 support is added. I'll update it.

Reply:
I see. I think we could set `kokkos@3.6:` then, instead of `kokkos@:3.7`. I was more worried about the lack of lower bound. In the next version however, we'll use `InitializationSettings` which appeared in Kokkos 3.7.